docs/combine.rst

```diff
@@ -0,0 +1,59 @@
+combine
+=======
+
+.. dfhack-tool::
+    :summary: Combine stacks of food and plants.
+    :tags: fort productivity stockpiles items plants
+
+Usage
+-----
+
+::
+
+    combine <all|here> [--dryrun] [--types=<comma separated list of types>]
+
+Examples
+--------
+``combine``
+    Displays help
+``combine all --dryrun``
+    Preview stack changes for all types in all stockpiles.
+``combine all``
+    Merge stacks for all stockpile and all types
+``combine all --types=meat,plant``
+    Merge ``meat`` and ``plant`` type stacks in all stockpiles.
+``combine here``
+    Merge stacks in stockpile located at game cursor.
+
+Commands
+-----------------
```

Comment:
```suggestion
--------
```

Reply:
rst requires that underlines match the header length

Reply:
done